File: week01/problem-set1/02-cash/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    float change;
    float m25 = 0.25;
    float m10 = 0.10;
    float m5 = 0.5;
    float m1 = 0.1;

    do{
        change = get_float("Change owed: ");
    }
    while(change <= 0);

    if(change % m25 == 0){
        
    }
}

File: week06/pset6/sentimental-cash/cash.py
from cs50 import get_float

def how_much(cents, coin_type):
    return cents // coin_type  # Usar // para obter a parte inteira

def main():
    while True:
        change = get_float("Change owed: ")
        if change >= 0:
            break

    quarters = how_much(change, 25)
    change -= quarters * 25

    dimes = how_much(change, 10)
    change -= dimes * 10

    nickels = how_much(change, 5)
    change -= nickels * 5

    pennies = how_much(change, 1)
    change -= pennies

    total = quarters + dimes + nickels + pennies
    print(total)

main()
